### ISSUE TITLE
PSX font: hand cursor overlap

### ISSUE THREAD

**Issue by jbaiter** [Albeoris/Memoria#790]
In dialog boxes that have a cursor, the cursor's x-offset is too high, making the cursor overlap with the rendered text.

![377840_20240825213202_1](https://github.com/user-attachments/assets/51be82c8-67a2-4e50-9c32-072b90383fb1)

**Comment by barkermn01** [comment]
It looks like you are but could i confirm are you using Moguri, are you using a none 4:3 resolution?

**Comment by jbaiter** [comment]
> It looks like you are but could i confirm are you using Moguri, are you using a none 4:3 resolution?

Yes, Moguri Mod with the widescreen toggle in the launcher enabled.
